setup.py

```diff
@@ -124,6 +124,17 @@ def getCommandLineOptions():
      "",
      "(optional) Skip nupic.core version comparison"]
   )
+  optionsDesc.append(
+    ["optimizations-native",
+    "value",
```

Comment:
@david-ragazzi how do I make the parameter-less options? I tried just letting the second line be `""` but it didnt work